packages/core/src/component.ts

```diff
@@ -3,25 +3,31 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Constructor, Provider, BoundValue} from '@loopback/context';
-import {Application} from '.';
-
-// tslint:disable:no-any
+import {Constructor, Provider, BoundValue, Context} from '@loopback/context';
```

Comment:
is `Context` import used in this file?

Reply:
Nope! Good catch.

Side-note: We *REALLY* need to add no-unused-imports to our lint rules.